### PR TITLE
added_tailwind+extra_colors

### PR DESCRIPTION
**Pull request by MaksymilianKeller**
Dodałem tailwind i kilka dodatkowych kolorów. W character.ejs trochę poeksperymentowałem i stworzyłem parę pomysłów na stylizację tekstu i ramek w grze.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,27 +1,44 @@
...
-}
+{
+  "name": "browser-game",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "devStart": "nodemon ./src/server.ts",
+    "start": "node ./build/server.js",
+    "build": "tsc",
+    "build-css": "postcss src/tailwind.css -o public/stylesheets/tailwind.css"
```
Comment (kkulewski) [inline package.json:9]: you can change it to run both commands:
`"build": "tsc && postcss src/tailwind.css -o public/stylesheets/tailwind.css",`

**Review by bartlomiejmont — APPROVED** [review]

**Comment by kkulewski** [comment]
@bartlomiejmont 

Maybe we should move:
- `views` from `/` to `/src/` (path: `src/views/`)
- `stylesheets` from `/public` to `/src/public` (path: `src/public/stylesheets`) 
and make required adjustmens? Right now we do copy everything to build anyway.

Then, in app.ts we can change:
- `app.set('views', 'build/views');` to `app.set('views', './views')` 
- `app.use(express.static('build/public'));` to `app.use(express.static('./public'));`

File: src/app.ts [hunk 2]
```diff
@@ -8,9 +8,9 @@ export const app = express();
 import { router }  from './routes/index';
 
 // view engine setup
-app.set('views', path.join(__dirname, '../views'));
+app.set('views', 'build/views');
```
Comment (kkulewski) [inline src/app.ts]: this doesn't feel right, I think it should be relative path like `'/views'`

File: src/app.ts [hunk 3]
```diff
@@ -8,9 +8,9 @@ export const app = express();
 import { router }  from './routes/index';
 
 // view engine setup
-app.set('views', path.join(__dirname, '../views'));
+app.set('views', 'build/views');
 app.set('view engine', 'ejs');
-app.use(express.static(path.join(__dirname, '../public')));
+app.use(express.static('build/public'));
```
Comment (kkulewski) [inline src/app.ts]: same, more like `'/public`

**Review by kkulewski — CHANGES_REQUESTED** [review]

**Review by kkulewski — APPROVED** [review]

**Pull request merged**